src/tools/python/output.ts

```diff
@@ -38,21 +38,22 @@ export class PythonToolOutput extends ToolOutput {
   }
 
   getTextContent() {
-    const fileList = this.outputFiles
-      .map((file) => `- [${file.filename}](${PythonToolOutput.FILE_PREFIX}:${file.id})`)
-      .join("\n");
-    return `The code exited with code ${this.exitCode}.
-stdout:
-\`\`\`
-${this.stdout}
-\`\`\`
-
-stderr:
-\`\`\`
-${this.stderr}
-\`\`\`
-
-${fileList ? "Files that were created or modified:\n" + fileList : "No files were created or modified."}`;
+    return (
+      (this.exitCode === 0
+        ? "The code executed successfully.\n"
```

Comment:
Would you like nothing to be outputted if the code has been executed successfully? I assume you should print the stdout.